apps/web/src/pages/auth/components/useAcceptInvite.ts

```diff
@@ -12,15 +12,15 @@ import { errorMessage } from '../../../utils/notifications';
 
 export function useAcceptInvite() {
   const { setToken } = useAuthContext();
-  const navigate = useNavigate();
   const queryClient = useQueryClient();
+  const navigate = useNavigate();
 
   const { isLoading, mutateAsync, error, isError } = useMutation<string, IResponseError, string>((tokenItem) =>
     api.post(`/v1/invites/${tokenItem}/accept`, {})
   );
 
   const submitToken = useCallback(
-    async (token: string, invitationToken: string, refetch = false) => {
+    async (token: string, invitationToken: string, refetch = false, isSignUp = true) => {
```

Comment:
by default `isSignUp` should be `false`, we only have it set to `true` in one place - sign up form